adaptive_breakpoints/example/lib/main.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+import 'package:adaptive_breakpoints/adaptive_breakpoints.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
```

Comment:
It would be nice to have an example that uses ~3 breakpoint entries to render 3 different layouts

Reply:
Done